doc/guides/dts/index.rst

```diff
@@ -356,25 +362,31 @@ Example: Subset of DTS/YAML files for NXP FRDM K64F (Subject to Change)::
   dts/bindings/pinctrl/nxp,kinetis-pinmux.yaml
   dts/bindings/serial/nxp,kinetis-uart.yaml
 
-YAML definitions for device nodes
-*********************************
+YAML bindings for device nodes
+******************************
 
-Device tree can describe hardware and configuration, but it doesn't tell the
+Device tree describes hardware and configuration, but it doesn't tell the
 system which pieces of information are useful, or how to generate configuration
-data from the device tree nodes.  For this, we rely on YAML files to describe
-the contents or definition of a device tree node.
-
-A YAML description must be provided for every device node that is to be a source
-of information for the system.  This YAML description can be used for more than
-one purpose.  It can be used in conjunction with the device tree to generate
-include information.  It can also be used to validate the device tree files
-themselves.  A device tree file can successfully compile and still not contain
-the necessary pieces required to build the rest of the software.  YAML provides
-a means to detect that issue.
-
-YAML files reside in a subdirectory inside the common and architecture-specific
-device tree directories.  A YAML template file is provided to show the required
-format.  This file is located at::
+data from the device tree nodes.  For this, we rely on YAML binding files to
+describe the contents or definition of a device tree node and instruct how the
+extracted information should be formatted.
+
+A YAML description (a.k.a YAML binding) must be provided for every device node
```

Comment:
Do you need to introduce the term "YAML description" ?

Can we stick to "YAML binding" ? Or is there a difference?

Reply:
Actually YAML description is what it is, YAML bindings is how we call them.
Indeed, "a.k.a" makes it unclear. Let me reword it.